features/mbedtls/targets/TARGET_CRYPTOCELL310/aes_alt.h

```diff
@@ -27,23 +27,13 @@
 extern "C" {
 #endif
 
-
-struct sw_aes_context
-{
-    int nr;                     /*!<  number of rounds  */
-    uint32_t *rk;               /*!<  AES round keys    */
-    uint32_t buf[68];           /*!<  unaligned data    */
-    int sw_key_set;
-};
-
 typedef struct
 {
-    struct {
-        SaSiAesUserContext_t CC_Context;
-        SaSiAesEncryptMode_t CC_cipherFlag;
-        SaSiAesUserKeyData_t CC_KeyData;
-    } cc_aes_context;
-    struct sw_aes_context sw_context;
+
+    int keysize_nr;             /*number of rounds for SW use case, and key size for CC use case*/
```

Comment:
Yeah, there is a couple of lines where it does not make much sense for the CC. As I said, it is a minor suggestion, it works either way. I was just suggesting to reduce the differences between this code and mbed TLS.